test/e2e_test_policy_crd.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+#
+# Copyright 2022 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+echo '::group:: invalid policy: both glob and regex'
```

Comment:
Awesomesauce! So, if we don't hand code these files, but use a for loop here:
```
for i in `ls ./test/testdata/cosigned/invalid/`
do
  if kubectl create -f ./test/testdata/cosigned/invalid/$i ; then
  then
    echo $i failed when it should not have
    exit 1
  else $i rejected as expected
done
```

Or something like that. Then if new files are added to .../invalid/ directory, then we don't have to modify this file at all.

What do you think?

Reply:
Yeah that sounds good. Will make the change!